packages/eslint-config/index.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   parserOptions: {
-    ecmaVersion: 2020,
+    ecmaVersion: 2022,
   },
   extends: ["airbnb-base", "@qlik/eslint-config-base", "prettier"],
```

Comment:
is this really needed if it is extending eslint-config-base ?

Reply:
Nope, I think it can be removed actually. I'll clean these up.